packages/twenty-server/src/engine/utils/custom-class-validator/ForbiddenWords.ts

```diff
@@ -8,14 +8,28 @@ import {
 
 @ValidatorConstraint({ async: false })
 export class ForbiddenWordsConstraint implements ValidatorConstraintInterface {
```

Comment:
Forbidden words was a bit misleading to me as I assume that it would forbid any string that contains this words. Should it be something more generic like `@IsNot(...)`  ?

---
After I added the above comment, I realized that class-validator already has @IsNotIn and @Matches ; why not use those?

Reply:
I missed the validator `IsNotIn` but to validate the pattern `api-*` we need a regex and the validator `IsNotIn` don't support it. So the custom validator is needed for that.

Reply:
Moreover the custom validator lowercase the value.

Reply:
Ok! Can we just rename it? Thanks!

Reply:
Done ✅